### ISSUE TITLE
Running Install-BoxstarterPackage against a remote computer does not allow sourcing a package from a custom NuGet Repo

### ISSUE THREAD

**Issue by mwrock** [chocolatey/boxstarter#10]
Migrated from https://boxstarter.codeplex.com/workitem/73

I have my system bootstrap package located in a custom NuGet repo and am running the following command's from my workstation PC against the server I am trying to BootStrap

```
#################
# Workstation PC
#################

# Set NuGet Repo
Set-BoxStarterConfig -NugetSources "http://www.myget.org/F/MyNugetFeed/api/v2;http://chocolatey.org/api/v2;http://10.4.48.112:8080/nuget/MyRepo"

# Start Remote Install
Install-BoxstarterPackage -ComputerName 10.136.175.171 -Credential $cred -PackageName MyPackageInMyCustomRepo -Verbose

```

The console then logs the following (I am only showing the important parts)

```
VERBOSE: [MyServer]Boxstarter: Installing 1 packages from
C:\Users\AdminAccount\AppData\Local\Temp\Boxstarter\BuildPackages;http://chocolatey.org/api/v2;http://www.myget.org/F/boxstarter/api/v2

.
.
.

Chocolatey (v0.9.8.23) is installing 'MyPackageInMyCustomRepo' and dependencies. By installing you accept the license for
'MyPackageInMyCustomRepo ' and each dependency you are installing.
Unable to find package 'MyPackageInMyCustomRepo'.

Command 'install' failed (sometimes this indicates a partial failure). Additional info/packages: MyPackageInMyCustomRepo
VERBOSE: [MyServer]Boxstarter: Exit Code: 1
Chocolatey reported an unsuccessful exit code of 1
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-Error
    + PSComputerName        : 10.136.175.171
```

When BoxStarter is doing this remote installation, it creates the temporary directory **C:\Users\MyUserName\AppData\Local\Temp\boxstarter** on the destination PC. Inside this folder is **boxstarter.config**. It is using this file to know where its NuGet soruces are. This is where the problem lies.

I tried putting **boxstarter.config** on the destination server there first with my custom NuGet repo, but when you run the **Install-BoxstarterPackage** against the remote server, this file is replaced.

I have also tried pre-installing BoxStarter on the destination server and running **Set-BoxStarterConfig** but when you run **Install-BoxstarterPackage** it will only look at the configuration from the **C:\Users\MyUserName\AppData\Local\Temp\boxstarter** folder.

I am not sure if this is by design or not, but if an additional parameter could be added to the **Install-BoxstarterPackage** function, for example **-UseLocalBoxstarterConfig**, this would resolve it. I guess the way to implement is, after sending the remote computer the **Boxstarter.zip** package and extracting it, also sends the workstations **boxstarter.config** file to the remote machine, so when the remote machine attempts to search for the package, it uses the same sources as the workstation machine.

**Comment by snigge** [comment]
I'm still having this issue and it doesn't seem to help to edit the config inside the zip file. Is this being worked on? The "remote installation" part of boxstarter is very flawed when this bug is present.

Keep up the good work!

**Comment by sshiman** [comment]
having the same problem...
I'm on a PS session (on a remote - Azure - computer) by using 

$rs = New-PSSession -ComputerName ([Uri]$Service.DNSName).DnsSafeHost -Port $RemotePS.Port -UseSSL:$true -Credential $cr -SessionOption (New-PSSessionOption -SkipCACheck) 
and then by invoking script block -

                      Invoke-Command -Session $rs -Scriptblock {                     
                            choco install boxstarter
                            choco install boxstarter.Azure
                            Set-ExecutionPolicy Unrestricted -Force
                            Import-Module Boxstarter.Chocolatey
                            Install-BoxstarterPackage -Package mssqlserver2012express 
                        }

getting this error -

MsSqlServer2012Express v11.00.3000.20140928
ERROR: The system cannot find the file specified.
    + CategoryInfo          : NotSpecified: (ERROR: The syst...file specified.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
    + PSComputerName        : eikostest09.cloudapp.net
 
NotSpecified: (:) [], RemoteException
SqlServer2012Express did not finish successfully. Boo to the chocolatey gods!
-----------------------
[ERROR] Unable to run scheduled task. Message from task was 
-----------------------
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-Error
    + PSComputerName        : xxxxxxxx.cloudapp.net
 
Package 'MsSqlServer2012Express v11.00.3000.20140928' did not install successfully: Unable to run scheduled task. Message from task was 
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Write-Error
    + PSComputerName        : xxxxxxx.cloudapp.net

Help! 
account is setup to have Admin rights. Does it require to setup some more rights somewhere?
This is definitely related to WinRM, 'cause I don't see such error while doing the same on the machine itself.

**Comment by pauby** [comment]
Duplicate of #6 and resolved in PR #320
